datasets/dataverkenner/zakelijkerechtenm/dataset.sql

```diff
@@ -1,4 +1,4 @@
-CREATE MATERIALIZED VIEW IF NOT EXISTS public.dataverkenner_zakelijkerechtenm_zakelijkerechten WITH (security_barrier) as
```

Comment:
waarom valt de security_barrier er af?

Reply:
Omdat het een materialized views is. 
https://www.postgresql.org/docs/current/sql-creatematerializedview.html